content/en/docs/concepts/services-networking/service.md

```diff
@@ -1173,6 +1173,38 @@ SCTP is not supported on Windows based nodes.
 The kube-proxy does not support the management of SCTP associations when it is in userspace mode.
 {{< /warning >}}
 
+## Application protocols
```

Comment:
nit / query
```suggestion
## Application protocol
```

Reply:
I'd consider making this a 3rd level heading that appears just before [#virtual-ips-and-service-proxies](https://kubernetes.io/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies) (so, part of the [Defining a Service](https://kubernetes.io/docs/concepts/services-networking/service/#defining-a-service) section).

It's already a long page and every extra second-level heading makes the table of contents take more vertical space.

@robscott, what do you think? This point is very open to discussion.

Reply:
That all makes a lot of sense, updated to match that now, thanks!